spec/controllers/api/orders_controller_spec.rb

```diff
@@ -181,5 +287,13 @@ def order_attributes
         :distributor_name, :special_instructions, :payment_capture_path
       ]
     end
+
+    def order_detailed_attributes
+      [
+        :number, :item_total, :total, :state, :adjustment_total, :payment_total,
```

Comment:
Do we need to list all of the attributes and test for them? It'll mean the spec has to change if the serializer is touched, but I don't see a real benefit.

Reply:
Especially since you subsequently test most of them directly and more explicitly...

Reply:
for external API endpoints like this one I think we should have these tests yes, ideally we would know what are the ones that are part of the public API (and cannot be removed). If the serializer changes we need to make sure these attributes remain in the payload as they are part of the public api.

I added a commit to remove repeated assertions :+1: